site2/website/blog/2022-02-24-apache-pulsar-community-welcomes-500th-contributor

```diff
@@ -0,0 +1,60 @@
+
+3. **Cluster Failover** [[PIP-121]](https://github.com/apache/pulsar/pull/13316) provides an automatic cluster level recovery mechanism that automatically redirects all Pulsar clients to a standby cluster in the event of catastrophic cluster failure of the active Pulsar cluster. This helps minimize the downtime experienced by the client applications by allowing them to seamlessly transition from one Pulsar to the next without user intervention.
+
+4. **Improved multi-tenancy support with tenant and namespace rate limiting.** [[PIP-82]](https://github.com/apache/pulsar/pull/11918) This feature allows users to configure a namespace or tenant wide limit for producers and consumers, and have that enforced irrespective of the number of topics in the namespace, allowing control of the max rate across a given namespace. 
+
+5. **Simplified microservices development** with Pulsar’s high-level APIs is now possible. [Exclusive producers](https://www.youtube.com/watch?v=Chw8VBS9WfI) enable developers to build coordination and consensus mechanisms.
+
+6. The new [Transactions feature](https://streamnative.io/blog/release/2021-06-14-exactly-once-semantics-with-transactions-in-pulsar/) empowers organizations to use Pulsar as a system-of-record that enables event-based architectures, similar to those popular in finance and real-time systems. With this update, Pulsar's message delivery/processing semantic evolved from message deduplication on a single topic, to atomic produce and acknowledgment over multiple topics. 
+
+7. [Function Mesh for Apache Pulsar](https://www.youtube.com/watch?v=fAn1HZdb50k) is now available. With Function Mesh, teams can better manage functions, connectors, and applications composed of multiple functions directly in Kubernetes.
+
+## What’s Next? 
+
+This year you can expect to see more exciting updates in the Pulsar project, including:
+
+1. [Moving Toward a ZookKeeper-Less Apache Pulsar](https://streamnative.io/blog/release/2022-01-25-moving-toward-a-zookeeperless-apache-pulsar/) Since its inception, Pulsar has used ZooKeeper as its distributed coordinator to store critical metadata information. Pulsar Improvement Plan [[PIP-45]](https://github.com/apache/pulsar/wiki/PIP-45%3A-Pluggable-metadata-interface) eliminates the ZooKeeper dependency and replaces it with a pluggable framework. This plug-and-play framework enables you to reduce the infrastructure footprint of Pulsar by leveraging alternative metadata and coordination systems based upon your deployment environment, resulting in improved consistency, resilience, stability, and reducing the technical debt of deployment. The end result is that starting in version 2.10, you can now run Pulsar without ZooKeeper. 
```

Comment:
Again you are referring to a SN blog post which does not link back in any way to the Apache Pulsar project.